test/check-machines-hostdevs

```diff
@@ -146,7 +148,7 @@ class TestMachinesHostDevs(VirtualMachinesCase):
                 b.wait_not_present("#vm-subVmTest1-hostdevs-dialog")
 
             def add(self):
-                m.execute("virsh dumpxml subVmTest1 > /tmp/vmxml1")
+                self.run_admin("virsh dumpxml subVmTest1 > /tmp/vmxml1", connectionName)
```

Comment:
The reason for the failure is likely that debian/ubuntu has /tmp protections enabled (i.e. for world-writable files), or possibly the tests just run in the reverse order. If /tmp/vmxml1 already exists and is owned by root, you can't rewrite it as user. You need to remove the file first (as root), or use a per-test random directory such as `self.vm_tmpdir`.